### PR TITLE
create index page

### PR DESCRIPTION
**Pull request by freddykwan91**
I leave the CSS and JS for later

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -4,17 +4,20 @@
...
-  a {
+  a, .chart-title {
     color: #484848;
     text-decoration: none;
     font-size: 22px;
     font-weight: bolder;
     transition: all 0.25s ease;
   }
-  &:hover a {
+  &:hover .chart-title {
```
Comment (mirhamasala) [inline app/assets/stylesheets/components/_card.scss]: I'm not sure if it's good practice to use &:hover when on line 7 your applying css to two different elements. It's kinda confusing. Perhaps there's no need to declare the a tag here either as we will have to do that in a general place for the entire website.

File: app/assets/stylesheets/components/_index.scss [hunk 2]
```diff
@@ -1,3 +1,11 @@
 // Import your components CSS files here.
 @import "alert";
 @import "navbar";
+@import "card";
+
+h1 {
```
Comment (mirhamasala) [inline app/assets/stylesheets/components/_index.scss]: h1 font-size 100px?

File: app/assets/stylesheets/components/_index.scss [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+  font-size: 100px;
+}
+.no-chart-info-text {
```
Comment (mirhamasala) [inline app/assets/stylesheets/components/_index.scss]: no-chart-info-text? Can we pick a better class name maybe?

Comment (arnauddlc) [reply app/assets/stylesheets/components/_index.scss]: Agree about the h1, should be nested within .no-chart-info-text (assuming that's where it appears) so it does not interfer with another class

Comment (mirhamasala) [reply app/assets/stylesheets/components/_index.scss]: The part "selectors" on this page can be helpful: http://htmlcheatsheet.com/css/

**Pull request merged**